src/erasure-code/jerasure/CMakeLists.txt

```diff
@@ -70,7 +74,7 @@ if(INTEL_SSE)
   add_dependencies(ec_jerasure_sse3 ${CMAKE_SOURCE_DIR}/src/ceph_ver.h)
   target_link_libraries(ec_jerasure_sse3 crush ${EXTRALIBS})
   set_target_properties(ec_jerasure_sse3 PROPERTIES VERSION 2.0.0 SOVERSION 2
-    COMPILE_FLAGS ${SSE3_FLAGS})
+    COMPILE_FLAGS ${COMPILE_TARGETS_FLAGS})
```

Comment:
this doesn't look right to me. we only want to pass supported sse3 flags for this target, not all supported flags